chia/types/spend_bundle.py

```diff
@@ -55,7 +55,12 @@ def name(self) -> bytes32:
         return self.get_hash()
 
     def debug(self, agg_sig_additional_data=bytes([3] * 32)):
-        debug_spend_bundle(self, agg_sig_additional_data)
+        """This method uses a global return value from logging.getLogger"""
```

Comment:
Maybe go

```
def debug(self, agg_sig_additional_data=bytes([3] * 32), f=sys.stdin):
```

And then add

```
def log(self, *args, **kwargs):
    s = StringIO.StringIO()
    self.debug(s, *args, **kwargs)
    log.info(s.getvalue())
```

Reply:
That said, I'm not sure that setting up a logger here is appropriate.

Reply:
It's kinda not appropriate, but I'm stuck between requiring the user to pass in a logger, or having this come out of some random file descriptor depending on if it is run under pytest or not.